h2o-py/tests/testdir_algos/glm/pyunit_glm_act_vs_pred.py

```diff
@@ -23,10 +23,14 @@ def test_predicted_vs_actual_by_variable():
     fr["predict"] = predicted["predict"]
     fr["CAPSULE"] = training_data["CAPSULE"]
     manual = fr.group_by("DPROS").mean().get_frame()
-    manual.set_names(["DPROS"] + list(pva_pd.columns))
-    manual_pd = manual.as_data_frame().set_index("DPROS")
+    print(manual)
```

Comment:
Is this intentional or just forgotten?

Reply:
intentional, it helps with debugging when the test fails - and it is a small frame (so no harm imho)